web/build.gradle

```diff
@@ -28,10 +28,12 @@ dependencies {
     compile 'org.springframework.boot:spring-boot-starter-security'
     compile project(':data')
     testCompile 'org.springframework.boot:spring-boot-starter-test'
+    implementation 'com.github.papaya-agh:papaya-interface:IO2019TEAM-68-SNAPSHOT'
+    testCompile group: 'org.spockframework', name: 'spock-spring', version: '1.3-groovy-2.5'
```

Comment:
Może jak wszędzie używamy skróconej wersji to tu też tak zróbmy ;)

Reply:
Nie wszędzie, postgres w data też tak był :stuck_out_tongue: